src/main/java/org/jboss/threads/EnhancedQueueExecutor.java

```diff
@@ -337,13 +329,10 @@ private static final class RuntimeFields {
         private static final long queueSizeOffset;
 
         static {
-            int cacheLine = CacheInfo.getSmallestDataCacheLineSize();
-            if (cacheLine == 0) {
-                // guess
-                cacheLine = 64;
-            }
+            // this is fine for pretty much 32 and 64 bit x86 and ARM processors; see
+            // https://github.com/ziglang/zig/blob/0.13.0/lib/std/atomic.zig#L424-L434
             // cpu spatial prefetcher can drag 2 cache-lines at once into L2
-            int pad = cacheLine > 128 ? cacheLine : 128;
+            int pad = 128;
```

Comment:
@dmlloyd I think I can drop the pad (or set it to 0) in case the number of processors is 1 - which CAN happen with containers, wdyt?  Maybe not that worthy, but still...

Reply:
Sure, as long as we can be sure that `1` means `1` and not e.g. `2×½`.

Reply:
thanks, yep, I'll make sure - I still feel bad about my bug on jboss logger encoding :"(